doc/contributor/interop.md

```diff
@@ -53,7 +53,7 @@ Calls `new`, passing the arguments as you'd expect.
 
 ### `HAS_SIZE`
 
-Returns true if the object responds to `size`, except for `String`.
+Returns `true` only for `Array`.
```

Comment:
For consistency, I guess `GET_SIZE` should only accepts Array.
But it seems relatively harmless.